app/services/twilio_service.rb

```diff
@@ -12,7 +12,7 @@ def sms_verification_pin(app_name)
   end
 
   def send_welcome_msg
-    user_name = User.current_user&.full_name
+    user_name = User.current_user&.full_name.presence
```

Comment:
👍

Reply:
interesting concept. not seen this before. like it

Reply:
Yep, `presence` method checks for the value and returns that value. and if there no value or its blank, it returns nil :)
Rails is amazing it aways excites me @steveyken ;)